packages/rule-no-vulnerable-javascript-libraries/src/rule.ts

```diff
@@ -222,7 +223,7 @@ const rule: IRuleBuilder = {
             },
             type: 'object'
         }],
-        worksWithLocalFiles: false
+        scope: Scope.site
```

Comment:
Is there a reason why this cannot be `any` (or at least a TODO)?

Reply:
I just change the current value to its equivalent, but you are right, after we add jsdom to the local connector, this should work with local file. Changing it to any!

Reply:
This should be the same as the `axe` rule. You have a comment in there so maybe just have that comment for now. I think the part to inject JS is going to be a bit more complicated in the local connector.